jax/experimental/jax2tf/tests/primitives_test.py

```diff
@@ -461,6 +455,15 @@ def test_stop_gradient(self):
     f = jax2tf.convert(lax.stop_gradient)
     self.assertEqual(f(tf.ones([])), 1.)
 
+  # test_bfloat16 checkes that https://github.com/google/jax/issues/3942 is fixed
+  def test_bfloat16(self):
+    def jax_fn(x):
+      x = x.astype(jnp.bfloat16)
+      x *= 2.
+      return x
+
+    tf_fn = jax2tf.convert(jax_fn)
+    self.assertEqual(tf_fn(tf.convert_to_tensor(1.375)), 2.750)
```

Comment:
I think that we should add another test where the JAX code uses a constant nd.array(jnp.bfloat16). That is legal in a JAX program.

Reply:
`test_bfloat16_constant` has been augmented with such code